source/class/cv/TemplateEngine.js

```diff
@@ -341,7 +348,16 @@ qx.Class.define('cv.TemplateEngine', {
         loader.addListenerOnce('stylesLoaded', this.generateManifest, this);
       }
       // load structure-part
-      await this.loadParts([cv.Config.getStructure()]);
+      try {
+        await this.loadParts([cv.Config.getStructure()]);
+      } catch (e) {
+        // Note: the timeout can be changed by the not published URL parameter
+        // timeoutStructureLoad for debugging reasons. Usually, the server
+        // must be quick enough so that the client doesn't run into any issues
+        // here.
+        this.__showFatalError(`${qx.locale.Manager.tr('loadParts "Structure" failed')}: ${e}`);
+        throw new Error('loadParts "Structure" failed');
```

Comment:
this seems redundant. in a non debug build the global error handler might also capture the thrown exception. I think it also created a error notification for that an we have it twice.

Reply:
I wanted to stop the program flow here as it doesn't make sense to continue